pytest.ini

```diff
@@ -9,3 +9,5 @@ filterwarnings =
   ignore:(?s).*one of the inputs that requires gradient is not of double precision.*:
   ignore:(?s).*Distutils was imported before Setuptools*:
   ignore:(?s).*To copy construct from a tensor, it is recommended to use*:
+  ignore:(?s).*pkg_resources is deprecated*:
+  ignore:(?s).*Deprecated call to `pkg_resources.declare_namespace*:
```

Comment:
These were a couple warnings that were popping up on my machine. I couldn't figure out how to resolve these, but it doesn't seem like anything we're doing, specifically, so I added exceptions.